DEVEL.md

```diff
@@ -202,14 +202,14 @@ Update metadata.annotations.containerImage and metadata.annotations.createdAT fi
 deployments/operator/manifests/bases/intel-device-plugins-operator.clusterserviceversion.yaml
 to match current operator version and current date
 
-Generate package manifests with:
+Fork the [Community Operators](https://github.com/k8s-operatorhub/community-operators) repo and clone it:
 ```
-$ make packagemanifests OPERATOR_VERSION=0.X.Y
+$ git clone https://github.com/<GitHub Username>/community-operators
 ```
 
-Copy the generated files to the Community Operators repo:
+Generate package manifests with:
 ```
-$ cp -r packagemanifests/* community-operators/operators/intel-device-plugins-operator/
+$ make packagemanifests OPERATOR_VERSION=0.X.Y PACKAGEMANIFESTS_DIR=/community-operators/operators/intel-device-plugins-operator/
```

Comment:
```suggestion
$ make packagemanifests OPERATOR_VERSION=0.X.Y PACKAGEMANIFESTS_DIR=community-operators/operators/intel-device-plugins-operator/
```

Reply:
PACKAGEMANIFESTS_DIR=community-operators/operators/intel-device-plugins-operator/ is a long line. Can we drop it?

Reply:
@bart0sh Ok, I just dropped the whole part.!